nav2_simple_commander/nav2_simple_commander/robot_navigator.py

```diff
@@ -557,7 +557,7 @@ def _getPathThroughPosesImpl(self, start, goals, planner_id='', use_start=False)
 
         goal_msg = ComputePathThroughPoses.Goal()
         goal_msg.start = start
-        goal_msg.goals = goals
+        goal_msg.goals.poses = goals
```

Comment:
Should we populate the header stamp/frame?

Reply:
sure but the stamp isn't really loyal to its definition if I set it to 'now'. I would need to pass the time the goals were computed in the function and that seems unnecessary currently

Reply:
This is the "simple" commander afterall, its not perfect in any regard :-)